tracer/src/Datadog.Tracer.Native/cor_profiler.h

```diff
@@ -195,7 +195,7 @@ class CorProfiler : public CorProfilerBase
     // Register Aspects into Dataflow
     //
     int RegisterIastAspects(WCHAR** aspects, int aspectsLength);
-
+    void GetIastMetrics(int* instrumentedSources, int* instrumentedPropagations, int* instrumentedSinks);
```

Comment:
I'd rather not modify the cor_profiler. This could be a stand alone function in `dataflow`.